src/icrar/leap-accelerate/tests/algorithm/PhaseRotateTests.cc

```diff
@@ -405,15 +405,15 @@ namespace icrar
 
             std::vector<casacore::MVDirection> directions =
             {
-                casacore::MVDirection(-0.4606549305661674,-0.29719233792392513),
-                casacore::MVDirection(-0.753231018062671,-0.44387635324622354),
-                casacore::MVDirection(-0.6207547100721282,-0.2539086572881469),
-                casacore::MVDirection(-0.41958660604621867,-0.03677626900108552),
-                casacore::MVDirection(-0.41108685258900596,-0.08638012622791202),
-                casacore::MVDirection(-0.7782459495668798,-0.4887860989684432),
-                casacore::MVDirection(-0.17001324965728973,-0.28595644149463484),
-                casacore::MVDirection(-0.7129444556035118,-0.365286407171852),
-                casacore::MVDirection(-0.1512764129166089,-0.21161026349648748)
+                casacore::MVDirection(-0.4606549305661674,-0.29719233792392513)
```

Comment:
I think you can simply do `... directions = {{-0.460..., -0.2971...}}` and avoid repeating the type.

Reply:
The confusion I have with this class is knowing for sure when it's being treated as a 2D vector or a 3D vector. I mostly end up repeating types to handle cases where the container template gets changed to an interface, however considering how rare that ends up happening I don't mind either way